BlamLib/BlamLib/Blam/Halo2/Tags/Code/Objects.cs

```diff
@@ -12,7 +12,7 @@ namespace BlamLib.Blam.Halo2.Tags
 	#region unit_seat_accerleration_struct_block
 	partial class unit_seat_acceleration_struct
 	{
-		void invert_value(ref float value)
+		void Invert_value(ref float value)
```

Comment:
Method names seems to be PascalCase not snake_case but that can be fixed some other time.